userspace/falco/CMakeLists.txt

```diff
@@ -194,6 +194,24 @@ if(MUSL_OPTIMIZED_BUILD AND CMAKE_BUILD_TYPE STREQUAL "release")
 	)
 endif()
 
+if(CMAKE_BUILD_TYPE STREQUAL "relwithdebinfo" AND NOT WIN32)
```

Comment:
So, what happens on win32 here?

Reply:
We'll have to figure out how to do the equivalent of objcopy in Windows if we want to separate out the debug info properly. I'm only familiar with one way of doing so, and it only works with Visual Studio so I'd like to come up with a more generic method.

Reply:
Thanks for the insight! For now it is not a big deal since we don't ship windows artifacts (ie: Falco is not officially distributed for windows; we only want to make sure that we build there), perhaps we can leave a TODO comment there?